src/sidebar/panel.js

```diff
@@ -205,6 +210,7 @@ function reconnectSync () {
   isAuthenticated = false;
   setAnimation(false, true, true); // animateSyncIcon, syncingLayout, warning
   savingIndicator.textContent = browser.i18n.getMessage('reconnectSync');
+  savingIndicator.style="cursor:pointer;";
```

Comment:
If it works for all sidebar actions, why put the cursor only for reconnect to sync?

Reply:
Because the task for that text field only.